src/common/Network/PacketDef/Zone/ClientZoneDef.h

```diff
@@ -323,6 +323,16 @@ struct FFXIVIpcMarketBoardRequestItemListingInfo :
   /* 0000 */ uint32_t requestId;
 };
 
+struct FFXIVIpcFreeCompanyUpdateShortMessageHandler :
+  FFXIVIpcBasePacket< FreeCompanyUpdateShortMessageHandler >
+{
+  char shortMessage[104]; // Contains old/gibberish data in remaining slots if not filled
```

Comment:
that's fine, the actual message is null terminated so you can ignore anything past it, gibberish or not